manifests/params.pp

```diff
@@ -122,7 +133,7 @@
   # Owner of the environments dir: for cases external service needs write
   # access to manage it.
   $server_environments_owner   = $user
-  $server_environments_group   = 'root'
+  $server_environments_group   = $root_group
```

Comment:
FWIW, if you don't want to do `undef`, then I agree that `Administrators` would be the next best thing.